rubygems-update.gemspec

```
@@ -1,8 +1,12 @@
 # frozen_string_literal: true
 
+version = File.foreach(File.join(__dir__, "lib", "rubygems.rb")) do |line|
+    /^\s*VERSION\s*=\s*"(.*)"/ =~ line and break $1
+end
```

We can simplify this:

```suggestion
version = File.read(File.join(__dir__, "lib", "rubygems.rb"))[/^\s*VERSION\s*=\s*"(.*)"/, 1]
```